fabrica.sh

```diff
@@ -25,15 +32,63 @@ Usage:
     Upgrades and instantiates chaincode on all relevant peers. Chaincode directory is specified in Fabrica config file.
 
   fabrica.sh [help | --help]
-    Prints the manual."
+    Prints the manual.
+
+  fabrica.sh updates
+    Prints all newer versions available.
+
+  fabrica.sh updateTo <version>
+    Updates Fabrica to specified version."
 }
 
 printVersion() {
   optional_full_flag=$1
   docker run -it --rm \
     -u "$(id -u):$(id -g)" \
     -v $(pwd):/network/target \
-    fabrica sh -c "/fabrica/docker-entrypoint.sh version $optional_full_flag"
+    $FABRICA_IMAGE sh -c "/fabrica/docker-entrypoint.sh version $optional_full_flag"
+}
+
+printUpdates() {
+    docker run -it --rm \
+    -u "$(id -u):$(id -g)" \
+    -v $(pwd):/network/target \
+    $FABRICA_IMAGE sh -c "/fabrica/docker-entrypoint.sh check-updates"
+}
+
+updateTo() {
+  version=$1
+
+  if [ -z "$version" ]; then
+    echo "Please specify version number, ie:"
+    echo "fabrica.sh updateTo 0.1.0"
+    exit 1
+  fi
+
+  sudo curl -Lf https://github.com/softwaremill/fabrica/releases/download/"$version"/fabrica.sh -o /usr/local/bin/fabrica.sh && sudo chmod +x /usr/local/bin/fabrica.sh
```

Comment:
`/usr/local/bin/fabrica.sh` jest ryzykowne, przecież różni ludzie mogą to trzymać w różych miejscach (w `opt`, albo w samym `bin`). imho powinien tutaj być podmieniany bieżący plik